src/PIL/TiffImagePlugin.py

```diff
@@ -1568,7 +1570,15 @@ def _save(im, fp, filename):
     if format != 1:
         ifd[SAMPLEFORMAT] = format
 
-    ifd[PHOTOMETRIC_INTERPRETATION] = photo
+    if PHOTOMETRIC_INTERPRETATION not in ifd:
+        ifd[PHOTOMETRIC_INTERPRETATION] = photo
+    elif im.mode == "1" and ifd[PHOTOMETRIC_INTERPRETATION] == 0:
+        inverted_im = im.copy()
+        px = inverted_im.load()
+        for y in range(inverted_im.height):
+            for x in range(inverted_im.width):
+                px[x, y] = 0 if px[x, y] == 255 else 255
```

Comment:
This only handles bilevel, but grayscale (mode "L") images can also have WhiteIsZero interpretation, so might be an idea to cover them as well here (maybe use 255-px[x ,y] and also check for "L" above?)